packages/theming/apple-theme/src/fluentAppleColors.macos.ts

```diff
@@ -0,0 +1,490 @@
+      light: '#FFFFFF',
+      dark: '#555555',
+    }),
+    neutralBackground3: DynamicColorMacOS({
+      light: '#000000',
+      dark: '#555555',
+    }),
+    neutralForeground2: '#000000',
+    neutralForeground3: DynamicColorMacOS({
+      light: '#272727',
+      dark: '#FFFFFF',
+    }),
+    neutralInverted: '#FFFFFF',
+    neutralStroke2: '#000000',
+  };
+}
+
+export function getStockAppleLightPalette(): ApplePalette {
+  return {
+    blue10: '#4F6BED',
+    blueMagenta20: '#8764B8',
+    blueMagenta30: '#5C2E91',
+    communicationBlue: '#0078D4',
+    communicationBlueShade10: '#106EBE',
+    communicationBlueShade20: '#005A9E',
+    communicationBlueShade30: '#004578',
+    communicationBlueTint10: '#2B88D8',
+    communicationBlueTint20: '#C7E0F4',
+    communicationBlueTint30: '#DEECF9',
+    communicationBlueTint40: '#EFF6FC',
+    cyan20: '#038387',
+    cyan30: '#005B70',
+    cyanBlue10: '#0078D4',
+    cyanBlue20: '#004E8C',
+    dangerPrimary: '#D92C2C',
+    dangerShade10: '#C32727',
+    dangerShade20: '#A52121',
+    dangerShade30: '#791818',
+    dangerTint10: '#DD4242',
+    dangerTint20: '#E87979',
+    dangerTint30: '#F4B9B9',
+    dangerTint40: '#F9D9D9',
+    gray20: '#69797E',
+    gray25: '#F8F8F8',
+    gray30: '#7A7574',
+    gray40: '#393939',
+    gray50: '#F1F1F1',
+    gray100: '#E1E1E1',
+    gray200: '#C8C8C8',
+    gray300: '#ACACAC',
+    gray400: '#919191',
+    gray500: '#6E6E6E',
+    gray600: '#404040',
+    gray700: '#303030',
+    gray800: '#292929',
+    gray900: '#212121',
+    gray950: '#141414',
+    green10: '#498205',
+    green20: '#0B6A0B',
+    magenta10: '#C239B3',
+    magenta20: '#881798',
+    magentaPink10: '#E3008C',
+    orange20: '#CA5010',
+    orange30: '#8E562E',
+    orangeYellow20: '#986F0B',
+    pinkRed10: '#750B1C',
+    presenceAvailable: '#6BB700',
+    presenceAway: '#FFAA44',
+    presenceBlocked: '#C50F1F',
+    presenceBusy: '#C50F1F',
+    presenceDnd: '#C50F1F',
+    presenceOffline: '#8A8886',
+    presenceOof: '#B4009E',
+    presenceUnknown: '#8A8886',
+    red10: '#D13438',
+    red20: '#A4262C',
+    successPrimary: '#13A10E',
+    successShade10: '#11910D',
+    successShade20: '#0F7A0B',
+    successShade30: '#0B5A08',
+    successTint10: '#27AC22',
+    successTint20: '#5EC65A',
+    successTint30: '#A7E3A5',
+    successTint40: '#CEF0CD',
+    warningPrimary: '#FFD335',
+    warningShade10: '#E6BE30',
+    warningShade20: '#C2A129',
+    warningShade30: '#8F761E',
+    warningTint10: '#FFD94E',
+    warningTint20: '#FFE586',
+    warningTint30: '#FFF2C3',
+    warningTint40: '#FFF8DF',
+
+    brandBackgroundDisabled: '#252525',
+    brandForegroundDisabled: '#252525',
+    neutralBackground2: '#FFFFFF',
+    neutralBackground3: '#000000',
+    neutralForeground2: '#000000',
+    neutralForeground3: '#272727',
+    neutralInverted: '#FFFFFF',
+    neutralStroke2: '#000000',
+  };
+}
+
+export function getStockAppleDarkPalette(): ApplePalette {
```

Comment:
Why do we need separate functions for light and dark palettes? (unlike iOS)

Reply:
Ah... we don't need the light and dark palettes anymore. That was an intermediary step while I was debugging. They can be fully removed.